sync/patches/values/patch.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+repo_dir=$(git rev-parse --show-toplevel) ; readonly repo_dir
+script_dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd ) ; readonly script_dir
+
+cd "${repo_dir}"
+
+readonly script_dir_rel=".${script_dir#"${repo_dir}"}"
+
+set -x
+cp "${script_dir_rel}/values.yaml" ./helm/envoy-gateway/values.yaml
```

Comment:
I don't like copying `values.yaml` instead of a git patch. Is it too complicated?

Reply:
my issue with using diff is that makes changes more complicated. And yes, we are using diff in other places; but I guess that changes in the values will happen more often (?)

I have an idea that might work, but I'd like too try it in a follow-up PR.

Reply:
ok :+1: